<launch>
  <arg name="cam_image_topic" />

  <node pkg="acciobot_main" type="ItemPublisher.py" name="item_publisher" output="screen" ></node>

  <!-- roslaunch fetch_api ar_desktop.launch cam_image_topic:=mock_point_cloud -->
  <include file="$(find fetch_api)/launch/ar_desktop.launch">
    <arg name="cam_image_topic" value="$(arg cam_image_topic)" />
  </include>

  <include file="$(find fetch_api)/launch/move_group.launch" />

  <node pkg="perception" type="point_cloud_demo" name="point_cloud_demo" output="screen">
    <remap from="cloud_in" to="$(arg cam_image_topic)" />
  </node>

  <node pkg="acciobot_main" type="CollisionHandler.py" name="collision_handler" output="screen"></node>
  <node pkg="acciobot_main" type="backend.py" name="accio_backend" output="screen">
    <remap from="cloud_in" to="$(arg cam_image_topic)" />
  </node>
</launch>
